master/master.cfg

```diff
@@ -1450,13 +1444,15 @@ c['db'] = {
 
 # GitHub Integration
 
-# Only testbranch builders need to be considered here
-builders = [str(b.name) for b in c['builders'] if b.builder_type.purpose == Purpose.halide_testbranch]
-generator = BuildStartEndStatusGenerator(builders=builders,
+generator = BuildStartEndStatusGenerator(builders=[b.name for b in c['builders']],
```

Comment:
Why did we skip the filtering on purpose? Is there a place that shows status for non-pull-request builds?

Reply:
I don't know when / why that was added, but it must have been ~end of December.

Reply:
Yep, that's about right. (We did it 'unfiltered' for years; this was added in the vain hope that it would help improve the 'dropped notification' problem, which it didn't, but it seems reasonable in its own right. Do as you see fit.)